CHANGELOG.md

```diff
@@ -8,6 +8,109 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+| [Certificate Management KIT](https://eclipse-tractusx.github.io/docs-kits/category/certificate-management-kit)                                                    |      n/a       |                               [0.1.0](https://eclipse-tractusx.github.io/docs-kits/kits/Certificate%20Management%20Kit/Changelog)                               |
+
+### Known Knowns
+
+- Relevant components and interfaces developed considering Gaia-X Trust Framework - 22.10
+- Connection to an external Clearing House service for the entire Catena-X ecosystem is required. The Gaia-X compliance service should generally be capable of recognizing all companies that are able to register with the Catena-X Portal.
+Conformity to [Asset Administration Shell (AAS) API v3.0](https://industrialdigitaltwin.org/wp-content/uploads/2023/04/IDTA-01002-3-0_SpecificationAssetAdministrationShell_Part2_API.pdf) and bugfix v3.0.1 is given. Fine granular access control introduced for optional use.
+- This Tractus-X release does not include a Managed Identity Wallet (MIW). The development team decided on a temporary bridging solution, based on a commercial application, to test and ship Release 24.05. Without implementing this interim solution, participants will not be able to fully utilize this Tractus-X release in consequence. An updated and validated version of MIW is expected in Release 24.08.
+- The Tractus-X EDC has undergone significant changes in its Management API, communication with external services and internal APIs.
+- Documentation content (e.g. architecture, manuals etc) and folder structure have not explicitly been checked for this Release, as relevant TRGs are not yet in force. The vast majority of components are evolutions of previous Releases, with a thorough review undergone. This leads to a high confidence in the quality of existing documentation within the product repositories.
+- FOSS components developed and pre-tested to TRL6 with artificial test data only
+- No crosscheck of functionality in verification environment
+- No execution of Load-, Performance- or Penetration Tests
+- No assignment of Export Control Classification Numbers (ECCN) to FOSS components
+- GeoBlocking recommended for Operations (GBaaS)
+- Security concept (overall) recommended for Operations
+- Listed components in "Unchanged, untested" category were not verified in combination with the current Release package and are only compatible with Catena-X Standards as specified. Use at your own risk at this time.
+
+### Runtime
+
+- Tested on [Kubernetes](https://en.wikipedia.org/wiki/Kubernetes) versions: `1.27.3`
+- Tested with [PostgreSQL](https://en.wikipedia.org/wiki/PostgreSQL) versions: `15`
+  - Three exceptions apply for
+    - Country Risk
+    - Tractus-X EDC
+    - Golden Record Business Partner Number (BPN) Service
+
+    Those components violate TRG 5.07 as they operate on Postgresql DB Version 16.x (not 15), but have been successfully tested for the current Release.
```

Comment:
"
Overall the release has been tested on the following versions:
- Tested on [Kubernetes](https://en.wikipedia.org/wiki/Kubernetes) versions: `1.27.3`
- Tested with [PostgreSQL](https://en.wikipedia.org/wiki/PostgreSQL) versions: `15`

Those following components violate TRG 5.07 as they operate on Postgresql DB Version 16.x (not 15), but have been successfully tested for the current Release:
- Golden Record Business Partner Number (BPN Service)

Those components violate TRG 5.07 as they operate on Postgresql DB Version 14.x (not 15), but have been successfully tested for the current Release:
- Country Risk


Notes: Tractus-X EDC has been tested on both Postgresql DB Versions: 16.x and 15.x
"
I'm unsure about the following:
- product-name = 'knowledge-agents' (last synced 109 days ago) -> 16.x
- product-name = 'mso-test' (last synced 82 days ago) -> 14.x

Reply:
updated